examples/morpheusvm/tests/e2e/e2e_test.go

```diff
@@ -20,187 +20,82 @@ import (
+	flagVars     *e2e.FlagVars
+	genesisBytes []byte
+	factory      *auth.ED25519Factory
 )
 
-func init() {
-	flag.DurationVar(
-		&requestTimeout,
-		"request-timeout",
-		120*time.Second,
-		"timeout for transaction issuance and confirmation",
-	)
-
-	flag.StringVar(
-		&networkRunnerLogLevel,
-		"network-runner-log-level",
-		"info",
-		"gRPC server endpoint",
-	)
-
-	flag.StringVar(
-		&avalanchegoLogLevel,
-		"avalanchego-log-level",
-		"info",
-		"log level for avalanchego",
-	)
-
-	flag.StringVar(
-		&avalanchegoLogDisplayLevel,
-		"avalanchego-log-display-level",
-		"error",
-		"log display level for avalanchego",
-	)
-
-	flag.StringVar(
-		&gRPCEp,
-		"network-runner-grpc-endpoint",
-		"0.0.0.0:8080",
-		"gRPC server endpoint",
-	)
-	flag.StringVar(
-		&gRPCGatewayEp,
-		"network-runner-grpc-gateway-endpoint",
-		"0.0.0.0:8081",
-		"gRPC gateway endpoint",
-	)
-
-	flag.StringVar(
-		&execPath,
-		"avalanchego-path",
-		"",
-		"avalanchego executable path",
-	)
-
-	flag.StringVar(
-		&pluginDir,
-		"avalanchego-plugin-dir",
-		"",
-		"avalanchego plugin directory",
-	)
-
-	flag.StringVar(
-		&vmGenesisPath,
-		"vm-genesis-path",
-		"",
-		"VM genesis file path",
-	)
-
-	flag.StringVar(
-		&vmConfigPath,
-		"vm-config-path",
-		"",
-		"VM configfile path",
-	)
-
-	flag.StringVar(
-		&subnetConfigPath,
-		"subnet-config-path",
-		"",
-		"Subnet configfile path",
-	)
-
-	flag.StringVar(
-		&outputPath,
-		"output-path",
-		"",
-		"output YAML path to write local cluster information",
-	)
-
-	flag.StringVar(
-		&mode,
-		"mode",
-		"test",
-		"'test' to shut down cluster after tests, 'run' to skip tests and only run without shutdown",
-	)
-
-	flag.UintVar(
-		&numValidators,
-		"num-validators",
-		5,
-		"number of validators per blockchain",
-	)
-}
-
 func TestE2e(t *testing.T) {
 	ginkgo.RunSpecs(t, "morpheusvm e2e test suites")
 }
 
-var _ = ginkgo.BeforeSuite(func() {
+func init() {
+	flagVars = e2e.RegisterFlags()
 	require := require.New(ginkgo.GinkgoT())
 
 	// Load default pk
-	// Address: morpheus1qrzvk4zlwj9zsacqgtufx7zvapd3quufqpxk5rsdd4633m4wz2fdjk97rwu
+	prefundedAddrStr := "morpheus1qrzvk4zlwj9zsacqgtufx7zvapd3quufqpxk5rsdd4633m4wz2fdjk97rwu"
 	privBytes, err := codec.LoadHex(
 		"323b1d8f4eed5f0da9da93071b034f2dce9d2d22692c172f3cb252a64ddfafd01b057de320297c29ad0c1f589ea216869cf1938d88c9fbd70d6748323dbf2fa7", //nolint:lll
 		ed25519.PrivateKeyLen,
 	)
 	require.NoError(err)
 	priv := ed25519.PrivateKey(privBytes)
-	factory := auth.NewED25519Factory(priv)
-
-	// Read the VM config in, so it can be passed in as JSON.
-	// ANR sometimes takes the param as a file path or JSON string
-	// and other times only a JSON string.
-	vmConfigData, err := os.ReadFile(vmConfigPath)
+	factory = auth.NewED25519Factory(priv)
+
+	gen := genesis.Default()
+	gen.WindowTargetUnits = fees.Dimensions{18446744073709551615, 18446744073709551615, 18446744073709551615, 18446744073709551615, 18446744073709551615}
```

Comment:
What are these numbers based off of? They seem weirdly precise

Reply:
Good call out, added a comment here and for `MaxBlockUnits`